qttypes/src/qtcore/qsettings.rs

```diff
@@ -0,0 +1,183 @@
+use crate::internal_prelude::*;
+use crate::QString;
+
+cpp! {{
+    #include <QtCore/QSettings>
+    #include <QtCore/QString>
+}}
+
+cpp_class!(
+    /// Wrapper around [`QSettings`][class] class.
+    ///
+    /// [class]: https://doc.qt.io/qt-5/qsettings.html
+    #[derive(Default)]
+    pub unsafe struct QSettings as "QSettings"
```

Comment:
QSettings is a QObject and i'm not confortable having that in a cpp_class that can be moved.

I think i would do  `pub unsafe struct QSettings as "std::unique_ptr<QSettings>"`